_bookdown.yml

```diff
@@ -10,3 +10,5 @@ language:
   ui:
     edit: "Edit"
     chapter_name: "Chapter "
+header-includes:
```

Comment:
hier bitte nichts einfügen. Das hast du ja in der Preambel im Latex Ordner eingefügt das sollte reichen?